test/chain.js

```diff
@@ -33,7 +33,7 @@ describe('chain', function() {
         return r + a;
       };
     };
-    var bound = R.chain(h, f);
+    var bound = R.chain(f, h);
     // (>>=) :: (r -> a) -> (a -> r -> b) -> (r -> b)
     // h >>= f = \w -> f (h w) w
     eq(bound(10), (10 * 2) + 10);
```

Comment:
i wouldn't mind seeing the `chain(append, head)([1, 2, 3]) //=> [1, 2, 3, 1]` example tested here. For my money that is a very clear illustration of the behavior